### ISSUE TITLE
NoMissingTypeExpose reports types that have an alias that expose them

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/elm-review-common#1]
NoMissingTypeExpose is reporting false positives.

# SSCCE

```elm
module A exposing (Project, toString)

import Internal

type alias Project =
    Internal.Project

toString : Project -> String
toString project = ""
```

`Project` in the `toString` definition gets reported with a message like this:

```
NoMissingTypeExpose: Private type `Review.Project.Internal.Project` should be
exposed

8| toString : Project -> String
              ^^^^^^^
9| toString project =

Users of this module will not be able to annotate a value of this type if they
wanted to. You should expose this type or an alias of this type.
```

I expected it not to be reported, since we already kind of do through the local `Project` type.

I suggest that the behavior be changed, so that if a type alias that aliases the exact missing type is exposed, no error is reported.

I do wonder whether `Internal.Project -> String` should be reported though (likely with a message saying to use the local `Project`). I guess it depends if it becomes harder to read in the docs.

(cc @sparksp)

**Comment by dillonkearns** [comment]
In case it's useful, I'm encountering this problem as well. Here's the aliased type that's triggering the false positive: https://github.com/dillonkearns/elm-pages/blob/a13cffc2d895fa76ae6a79fcd5d6e6ed4b6913d2/src/DataSource/Glob.elm#L224-L225.

**Comment by jfmengels** [comment]
Fix released in 1.1.1 :)
